.github/workflows/push_tests.yml

```diff
@@ -71,12 +71,6 @@ jobs:
       - name: NVIDIA-SMI
         run: |
           nvidia-smi
-      - name: Tailscale
-        uses: huggingface/tailscale-action@v1
-        with:
-          authkey: ${{ secrets.TAILSCALE_SSH_AUTHKEY }}
-          slackChannel: ${{ secrets.SLACK_CIFEEDBACK_CHANNEL }}
-          slackToken: ${{ secrets.SLACK_CIFEEDBACK_BOT_TOKEN }}
```

Comment:
How would we know the IP address to SSH into without this step?

Reply:
The SSH access to a particular runner is only available for a certain amount of time. Practically speaking we might not have the time to actually debug anything while this access is still available (just due to having other obligations)

We have a dedicated workflow that allows us to SSH into a runner machine which is better for debugging since we can spin it up as needed instead of when the job completes

Reply:
Makes sense.